### PR TITLE
pure matlab sample

### PR DESCRIPTION
**Pull request by ForrestTrepte**
Demonstrates how to create a simulation via MATLAB code without using a Simulink model.

### CODE REVIEW

**Review by davidhcoe — COMMENTED** [review]
when you define the state/action schema with just the names, and then do your observation as:

 obj.observation1 = action.action_array1 * 2;
 
 that gets converted to the inkling array structure?

I would recommend removing the commented parts of the .ink file as well

**Review by davidhcoe — COMMENTED** [review]
re: bonsaiAssess -- you can run assessment using the bonsaiTrain file. I dont think the statement would be "its not supported" rather than "this file is not required to do so"

**Comment by ForrestTrepte** [comment]
@davidhcoe: Thanks for the feedback!
* Regarding state/action schema names these are sent to the Bonsai service as JSON. If they are arrays then the value of that variable will be serialized as an array. So it *seems* to work automatically.
* I will remove the inkling comments.
* I will reword the bonsaiAssess comment.

File: samples/pure-matlab/bonsaiExportConnect.m [hunk 1]
```diff
@@ -0,0 +1,5 @@
+% Copyright (c) Microsoft Corporation.
+% Licensed under the MIT License.
+
+% The export connect mode (viewing an exported brain running in Simulink)
+% is not supported in this pure MATLAB sample.
```
Comment (nmann4) [inline samples/pure-matlab/bonsaiExportConnect.m:5]: Not a now thing but you could leverage the ExportedBrainClient code to write a pure matlab export script.

It doesn't necessarily have to follow the convention of what the simulink samples do.

**Review by nmann4 — APPROVED** [review]
Code LGTM.

File: samples/pure-matlab/bonsaiExportConnect.m (hunk 1 above)
Comment (ForrestTrepte) [reply samples/pure-matlab/bonsaiExportConnect.m:5]: That's a good point--would be good future work to support that workflow.

**Pull request merged**